frontend/isoko/src/components/businessPage/BusinessSidebar.tsx

```diff
@@ -0,0 +1,194 @@
+import React from 'react';
+import styled from 'styled-components';
+import StyledButton from '../../styles/StyledButton';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+   faClock,
+   faMapMarkerAlt,
+   faLink,
+} from '@fortawesome/free-solid-svg-icons';
+import { useNavigate } from 'react-router-dom';
+
+interface SidebarContainerProps {
+   width?: string;
+}
+
+const SidebarContainer = styled.div<SidebarContainerProps>`
+   width: ${(props) => (props.width ? props.width : '300px')};
```

Comment:
How does this variable width work? Is this to make sure we have responsiveness

Reply:
Yep, this is just so we have a default width, but it can be updated through using styled-components on my component